src/ripple/overlay/impl/OverlayImpl.cpp

```diff
@@ -433,12 +433,19 @@ void
 OverlayImpl::setupValidatorKeyManifests (BasicConfig const& config,
                                          DatabaseCon& db)
 {
-    auto const loaded = manifestCache_.loadValidatorKeys (
+    auto loaded = manifestCache_.loadValidatorKeys (
+        config.section ("validators"),
+        journal_);
+
+    if (!loaded)
+        Throw<std::runtime_error> ("Unable to load keys from [validators]");
+
```

Comment:
Thanks @HowardHinnant I was thinking more about terse code than the allocations. I didn't want to write either `Throw<...>("Unable..." + std::string(section))`, so I converted to a string right away. Somehow the string in the loop bothers me less than the string in the function call. Shrug.